tests/db_engine_specs/bigquery_tests.py

```diff
@@ -57,3 +73,112 @@ def test_timegrain_expressions(self):
                 col=col, pdf=None, time_grain="PT1H", type_=type_
             )
             self.assertEqual(str(actual), expected)
+
+    def test_fetch_data(self):
+        """
+        DB Eng Specs (bigquery): Test fetch data
+        """
+        # Mock a google.cloud.bigquery.table.Row
+        class Row(object):
+            def __init__(self, value):
+                self._value = value
+
+            def values(self):
+                return self._value
+
+        data1 = [(1, "foo")]
+        with mock.patch.object(BaseEngineSpec, "fetch_data", return_value=data1):
+            result = BigQueryEngineSpec.fetch_data(None, 0)
+        self.assertEqual(result, data1)
+
+        row1 = Row(1)
+        row2 = Row(2)
+        data2 = [row1, row2]
```

Comment:
Perhaps just throw those on the same row: `data2 = [Row(1), Row(2)]`

Reply:
yes, better